scripts/skopeo-results.sh

```diff
@@ -15,13 +15,31 @@ function skopeo_inspect() {
         "${image}"
 }
 
-phase "Extracting '${PARAMS_SOURCE_IMAGE_URL}' source image digest"
-source_digest="$(skopeo_inspect "${PARAMS_SOURCE_IMAGE_URL}" "${PARAMS_SRC_TLS_VERIFY}")"
-phase "Source image digest '${source_digest}'"
+if [ -n "${PARAMS_SOURCE_IMAGE_URL}" ] && [ -n "${PARAMS_DESTINATION_IMAGE_URL}" ]; then
+    phase "Extracting '${PARAMS_SOURCE_IMAGE_URL}' source image digest"
+    source_digest="$(skopeo_inspect "${PARAMS_SOURCE_IMAGE_URL}" "${PARAMS_SRC_TLS_VERIFY}")"
+    phase "Source image digest '${source_digest}'"
 
-phase "Extracting '${PARAMS_DESTINATION_IMAGE_URL}' destination image digest"
-destination_digest="$(skopeo_inspect "${PARAMS_DESTINATION_IMAGE_URL}" "${PARAMS_DEST_TLS_VERIFY}")"
-phase "Destination image digest '${destination_digest}'"
+    phase "Extracting '${PARAMS_DESTINATION_IMAGE_URL}' destination image digest"
+    destination_digest="$(skopeo_inspect "${PARAMS_DESTINATION_IMAGE_URL}" "${PARAMS_DEST_TLS_VERIFY}")"
+    phase "Destination image digest '${destination_digest}'"
+else
```

Comment:
Before proceeding directly should we make sure whether workspace is provided or not

Reply:
While it is not needed (the task run will fail anyway with the message that url.txt file was not found), I have added to distinguish these 2 scenarios - when the workspace was provided but the file is not present and when the workspace wasn't provided at all.